### ISSUE TITLE
Exception on save (Flutter 1.17 + GeniusScanSDK 3.0.18)

### ISSUE THREAD

**Issue by aghanti7** [thegrizzlylabs/geniusscan-sdk-demo#26]
After upgrading to latest (3.0.18 + flutter v1.17), the scanner works, but I get the following exception in the logs:

```
E/WindowManager(24491): 	at androidx.fragment.app.Fragment.performResume(Fragment.java:2649)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:922)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
E/WindowManager(24491): 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:439)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManagerImpl.java:2079)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:1869)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:1824)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:1727)
E/WindowManager(24491): 	at androidx.fragment.app.FragmentManagerImpl$2.run(FragmentManagerImpl.java:150)
E/WindowManager(24491): 	at android.os.Handler.handleCallback(Handler.java:873)
E/WindowManager(24491): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/WindowManager(24491): 	at android.os.Looper.loop(Looper.java:193)
E/WindowManager(24491): 	at android.app.ActivityThread.main(ActivityThread.java:6898)
E/WindowManager(24491): 	at java.lang.reflect.Method.invoke(Native Method)
E/WindowManager(24491): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
E/WindowManager(24491): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

Steps:

1. Launch scanner 3-screen flow
2. On the 3rd screen, when you hit save, the pdf saves ok, but the above exception stack trace is seen in the logs.

**Comment by guillaume-tgl** [comment]
Thanks for reporting this issue. This shouldn't result in a crash since the error happens after an activity is finished but we will fix this anyway for the next version so that no error is displayed in the logs.

**Comment by guillaume-tgl** [comment]
Fixed in v3.0.19
